### PR TITLE
Correct a grammatical error

### PR DESCRIPTION
**Pull request by AllForNothing**
1. Modify `Skip to update` to `Skip updating`
Thank you for contributing to Harbor!

# Comprehensive Summary of your change

# Issue being fixed
Fixes #(issue)

Please indicate you've done the following:
- [x] Well Written Title and Summary of the PR
- [x] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [x] Made sure tests are passing and test coverage is added if needed.
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Review by OrlinVasilev — APPROVED** [review]
lgtm

**Review by YangJiao0817 — APPROVED** [review]

File: src/portal/src/i18n/lang/de-de-lang.json [hunk 1]
```diff
@@ -832,7 +832,7 @@
         "ROOT_CERT_LINK": "Download",
         "REGISTRY_CERTIFICATE": "Registry Zertifikat",
         "NO_CHANGE": "Speichern abgebrochen, da nichts verändert wurde",
-        "SKIP_SCANNER_PULL_TIME": "Skip to update pull time for scanner",
+        "SKIP_SCANNER_PULL_TIME": "Skip updating pull time for scanner",
```
Comment (OrlinVasilev) [inline src/portal/src/i18n/lang/de-de-lang.json:835]:
```suggestion
        "SKIP_SCANNER_PULL_TIME": "Retain image last pull time on scanning",
```

**Comment by OrlinVasilev** [comment]
@AllForNothing I think "Retain image last pull time on scanning" is better

File: src/portal/src/i18n/lang/de-de-lang.json [hunk 2]
```diff
@@ -855,7 +855,7 @@
             "HOURLY_CRON": "Läuft einmal am Anfang jeder Stunde. Identisch zu 0 0 * * * *.",
             "WEEKLY_CRON": "Läuft wöchentlich um Mitternacht zwischen Sa/So. Identisch zu 0 0 0 * * 0.",
             "DAILY_CRON": "Läuft täglich um Mitternacht. Identisch zu 0 0 0 * * *.",
-            "SKIP_SCANNER_PULL_TIME_TOOLTIP": "Skip to update pull time when pulling image from scanner."
+            "SKIP_SCANNER_PULL_TIME_TOOLTIP": "Skip updating pull time when pulling image from scanner."
```
Comment (OrlinVasilev) [inline src/portal/src/i18n/lang/de-de-lang.json:858]:
```suggestion
            "SKIP_SCANNER_PULL_TIME_TOOLTIP": "Vulnerability scanner(e.g. Trivy) will not update the image "last pull time" when scanning the image".
```

**Comment by AllForNothing** [comment]
@OrlinVasilev  PR updated and ready for review

**Review by OrlinVasilev — APPROVED** [review]
looks great, thanks!

**Pull request merged**